packages/govern-lib/src/utils/NetworkContext.ts

```diff
@@ -0,0 +1,96 @@
+import { Address, Networkish, Network } from '../types'
+import { networkContexts } from '../config'
+import { ErrorInvalidNetwork } from '../errors'
+
+class NetworkContext implements Network {
+  readonly chainId: number
+  readonly ensAddress: Address
+  readonly name: string
+  readonly subgraphUrl: string
+
+  static chainIdFromName(name: string): number | undefined {
```

Comment:
I think the ``constructor`` should always be on top of the class also if static factory methods do exist.

Reply:
I was trying to separate static methods from instance ones, but I don’t have any strong opinion, let’s do this :+1:

Reply:
I have probably sometimes a to strong opinion in such cases 🙈😜